src/Console/ChromeDriverCommand.php

```diff
@@ -311,10 +310,10 @@ protected function resolveChromeDriverDownloadUrl(string $version, string $os)
      */
     protected function getUrl(string $url)
     {
-        return Http::withOptions(array_merge([
+        return (new Client())->get($url, array_merge([
             'verify' => $this->option('ssl-no-verify') === false,
         ], array_filter([
             'proxy' => $this->option('proxy'),
-        ])))->get($url)->body();
+        ])))->getBody();
```

Comment:
There two issue with these changes if we want to replace HTTP Client:

* We need to handle none 2xx response and throw exception
* Response may return a stream instead of string, it best to cast the return value to string.

Reference: https://github.com/laravel/framework/blob/d3586ebfd88d1ac62035289d1172b7978cc8e429/src/Illuminate/Http/Client/PendingRequest.php#L872-L933

Reply:
For example: https://github.com/orchestral/dusk-updater/blob/f407d18d7134c4d29e387f00909222f12a230670/src/helpers.php#L55-L67

Reply:
@crynobone changes added as requested. The exception handling is the same as the download methods exception handling.